lib/node_modules/@stdlib/stats/base/dists/negative-binomial/ctor/benchmark/benchmark.js

```diff
@@ -33,15 +34,22 @@ var NegativeBinomial = require( './../lib' );
 
 bench( pkg+'::instantiation', function benchmark( b ) {
 	var dist;
+	var len;
 	var r;
 	var p;
 	var i;
 
+	len = 100;
+	r = new Float64Array( len );
+	p = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		r[ i ] = discreteUniform( 1.0, 50 );
```

Comment:
```suggestion
		r[ i ] = discreteUniform( 1, 50 );
```

This change is consistent throughout the PR.